CHANGELOG.md

```diff
@@ -1,17 +1,18 @@
 <!-- usage documentation: http://expeditor-docs.es.chef.io/configuration/changelog/ -->
 # Change Log
 
-<!-- latest_release 0.4.11 -->
-## [v0.4.11](https://github.com/chef/mixlib-archive/tree/v0.4.11) (2018-07-31)
+<!-- latest_release 0.4.12 -->
+## [v0.4.12](https://github.com/chef/mixlib-archive/tree/v0.4.12) (2018-08-01)
 
 #### Merged Pull Requests
-- Add github issue template &amp; standardize expeditor config [#26](https://github.com/chef/mixlib-archive/pull/26) ([tas50](https://github.com/tas50))
+- Update codeowners and add PR template [#27](https://github.com/chef/mixlib-archive/pull/27) ([tas50](https://github.com/tas50))
```

Comment:
You shouldn't update the changelog, that's handled automatically by Expeditor and it's not good practice to increment the version in a PR (that's for the maintainers to do)